src/package.rs

```diff
@@ -437,3 +450,65 @@ impl fmt::Debug for PackageId {
             .finish()
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
I think we should definitely keep this test, but i would prefer to move it to the `tests` directory as it does IO 🙂

Can we get around the oop `PackageStore` and `Cmd` structs for now and keep the testing aspect?

I do see value in being able to configure the directory eg. when you want to run commands in a nested dir / need to bubble up to the next manifest as in cargo workspaces but i think this feature is not so pressing right now to be worth the required complexity